frontend.py

```diff
@@ -399,7 +400,26 @@ def runtime(state):
 				clearConsole()
 				printAppUse()
 				state = 1
-
+	# Convert Notes
+			case 16: 
+				lineBreak(columns, getConfig(4))
+				printColor("What is the title of the note you would like to convert to MKDown?",getConfig(2), "")
+				lineBreak(columns,getConfig(4))
+				search_by = input(": ")
+				desired_response = ['content']
+				api_response = apiCalls.searchNotes('title', search_by, desired_response)
+				entries = translation(api_response.content)
+				parsing = entries.split("\n")
+				content = parsing[1].replace(r'\n', '\n')
+				title = parsing[0]
+				if ".mk" not in title: 
```

Comment:
markdown files have .md file extensions not .mk

Reply:
I fixed this